src/util.h

```diff
@@ -74,22 +74,10 @@ class CClient;  // forward declaration of CClient
 
 
 /* Global functions ***********************************************************/
-// converting double to short
-inline short Double2Short ( const double dInput )
+// range check audio samples
+static inline float clipFloat ( const float fInput )
 {
-    // lower bound
-    if ( dInput < _MINSHORT )
-    {
-        return _MINSHORT;
-    }
-
-    // upper bound
-    if ( dInput > _MAXSHORT )
-    {
-        return _MAXSHORT;
-    }
-
-    return static_cast<short> ( dInput );
+    return qBound ( -1.0f, fInput, 1.0f );
```

Comment:
This is _so_ simple I'd almost prefer it written out inline when used.  It saves wondering what the method does / how it's defined, etc.